### PR TITLE
Update bio_processes.json

### PR DESCRIPTION
**Pull request by buckly90**
Rebalanced cell metabolism to correct issues caused by variable tickrate.

### CODE REVIEW

**Comment by hhyyrylainen** [comment]
add to whitelist

File: scripts/simulation_parameters/microbe_stage/bio_processes.json [hunk 1]
```diff
@@ -160,7 +160,7 @@
         "inputs": {
             "carbondioxide": 0.09,
             "oxygen": 0.21,
-            "iron": 0.12
+            "iron": 0.024
```
Comment (GatoCreador887) [inline scripts/simulation_parameters/microbe_stage/bio_processes.json]: I don't think iron needs reducing, it already generated plenty of energy and iron is quite common.

Comment (buckly90) [reply scripts/simulation_parameters/microbe_stage/bio_processes.json]: You're right, It probably shouldnt be reduced nearly as much as the others, I will test that feature more specifically and see if it should be reduced as much, if at all.

**Comment by hhyyrylainen** [comment]
Is this bit of balancing done?

**Comment by buckly90** [comment]
> 
> 
> Is this bit of balancing done?

It is for now, though I would like for others to test it and tell me how it feels if possible. However if you think the changes are fine, feel free to merge. We can always further adjust the values if needed.

**Pull request merged**